packages/ckeditor5-dev-build-tools/tests/plugins/splitCss/fixtures/duplicated-definitions/file2.css

```diff
@@ -0,0 +1,8 @@
+:root {
+	--variable1: blue;
```

Comment:
I'd be good if this had different value from the `--variable1` in the `file1.css`. This way we could check if last definition of a given variable is used (the same as in CSS).